package.js

```diff
@@ -1,20 +1,20 @@
 Package.describe({
   name: "jazeee:spiderable-longer-timeout",
   summary: "Extended spiderable package: SSL, caching, longer timeout, no stdin issues, publish flag",
-  version: "1.2.8",
+  version: "1.2.9",
   git: "https://github.com/jazeee/jazeee-meteor-spiderable"
 });
 
 Package.onUse(function (api) {
   api.versionsFrom("METEOR@0.9.0");
-  api.use(['webapp', 'mongo@1.1.0', 'ostrio:meteor-root@1.0.0', 'sha', 'cmather:handlebars-server@0.2.0'], 'server');
```

Comment:
From what I can tell, the versions haven't changed.
On Sep 25, 2015 2:43 PM, "dr.dimitru" <notifications@github.com> wrote:

> In package.js
> <https://github.com/jazeee/jazeee-meteor-spiderable/pull/20#discussion_r40468330>
> :
>
> >    git: "https://github.com/jazeee/jazeee-meteor-spiderable"
> >  });
> >
> >  Package.onUse(function (api) {
> >    api.versionsFrom("METEOR@0.9.0");
> > -  api.use(['webapp', 'mongo@1.1.0', 'ostrio:meteor-root@1.0.0', 'sha', 'cmather:handlebars-server@0.2.0'], 'server');
>
> Hi @enricofoschi <https://github.com/enricofoschi> , that I was telling
> you yesterday.
> @jazeee <https://github.com/jazeee> could you just add latest vers?
> ostrio:meteor-root@1.0.0 is not changed
> Probably locally this error is mute, unless you're going to publish package
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jazeee/jazeee-meteor-spiderable/pull/20/files#r40468330>
> .
>